### PR TITLE
Added OK events

### PR DESCRIPTION
**Pull request by askpatrickw**

### CODE REVIEW

File: states/oklahoma/elections.toml [hunk 1]
```diff
@@ -7,13 +7,124 @@ original_date = 2020-03-03
...
+]
+
+[20200630.absentee]
+received_by = 2020-06-30T19:00:00
+in_person_start = 2020-06-25T08:00:00
+in_person_by = 2020-06-29T17:00:00
+sources = [
+    "https://www.ok.gov/elections/Voter_Info/Absentee_Voting/",
+    "https://www.ok.gov/elections/Election_Info/2020_Election_Calendar.html"
+]
```
Comment (tannewt) [inline states/oklahoma/elections.toml]: I think you want this section on the next two elections as well.

**Review by tannewt — CHANGES_REQUESTED** [review]
Thank you!

**Comment by askpatrickw** [comment]
Review again based on this info.


Returning Your Ballot

If your absentee ballot is mailed to you, you must return it to the county election board by mail. An absentee ballot must be received by the county election board before 7 p.m. on election day to be counted. Standard absentee ballots may also be delivered in-person to your county election board during regular business hours. Ballots delivered by hand must be returned no later than the end of business hours on the day before the election. Only standard absentee ballots may be returned in-person, and the voter must show proper identification. Those who request absentee ballots under the special conditions listed below for those who are physically incapacitated or care for physically incapacitated persons who cannot be left alone may only return their ballots by mail.

In-person Absentee Voting

Voters may cast an absentee ballot in person at the county election board office from 8 a.m. to 6 p.m. on Thursday and Friday before all elections. For state and federal elections only, in-person absentee voting also is available from 9 a.m. to 2 p.m. on Saturday. In-person absentee voters must fill out and sign an application form when they arrive to vote.



Also.... if any events _should_ be there (you think) but there's no info... add them and comment them out.

**Comment by askpatrickw** [comment]
Yep... I just totally missed adding those. Good catch, thank you !

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**